src/System.Device.Ports/System.Device.Ports.SerialPort/Interop/Unix/Interop.Unix.Libraries.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.Device.Ports.SerialPort
```

Comment:
As discussed during our call last week, I think it is better to just remove all of the PInvoke calls for now since I think it is best to start from the top (API design) to the bottom since I believe we will end up finding that our implementation will look very different to the one in System.IO.Ports given our API shape will also be different.